app/src/main/java/com/amaze/filemanager/filesystem/RootHelper.java

```diff
@@ -128,7 +128,7 @@ public static boolean isDirectory(String toTest, int count) throws ShellNotRunni
       for (String currentLine : resultLines) {
         if (contains(currentLine.split(" "), name)) {
           try {
-            HybridFileParcelable parsedFile = FileUtils.parseName(currentLine);
+            HybridFileParcelable parsedFile = FileUtils.parseName(currentLine, true);
```

Comment:
isStat is never false.

Reply:
Not always. There are two places that would cause `isStat` to become `false`:

https://github.com/TranceLove/AmazeFileManager/blob/feature/2263/app/src/main/java/com/amaze/filemanager/filesystem/root/ListFilesCommand.kt#L56-L64

https://github.com/TranceLove/AmazeFileManager/blob/feature/2263/app/src/main/java/com/amaze/filemanager/filesystem/RootHelper.java#L125-L127